synapse/storage/databases/main/deviceinbox.py

```diff
@@ -94,15 +94,22 @@ def __init__(
                     notifier=hs.get_replication_notifier(),
                     stream_name="to_device",
                     instance_name=self._instance_name,
-                    tables=[("device_inbox", "instance_name", "stream_id")],
+                    tables=[
+                        ("device_inbox", "instance_name", "stream_id"),
+                        ("device_federation_outbox", "instance_name", "stream_id"),
+                    ],
```

Comment:
Why don't we see a problem on postgres then?

Reply:
Because postgres uses a sequence in the DB to generate the IDs. This list of tables is only for enforcing consistency with that sequence at startup time.

Reply:
Oh yeah!